### PR TITLE
Bug Bash issues fixes

### PR DESCRIPTION
**Pull request by kcheekuri**
[Preview this branch](https://ms.portal.azure.com/?feature.phoenixendpoint=https://juno-test.documents-dev.windows-int.net&dataExplorerSource=https://cosmos-explorer-preview.azurewebsites.net/commit/27e3bf7a077fb8ccb5091e0a169e846cf158b0c9/explorer.html?feature.pr=https://github.com/Azure/cosmos-explorer/pull/1162#home)

This PR:
- Fixes BUG-1469584: New notebook creation issue under GitHub context menu.
- Fixes BUG-469713: Rename temporary workspace.

### CODE REVIEW

File: src/Explorer/Tree/ResourceTree.tsx [hunk 1]
```diff
@@ -397,6 +397,11 @@ export const ResourceTree: React.FC<ResourceTreeProps> = ({ container }: Resourc
       },
     ];
 
+    // For Temporary workspaces removing "Rename" in context menu.
```
Comment (srinathnarayanan) [inline src/Explorer/Tree/ResourceTree.tsx:400]: make it "//disallow renaming of temporary notebook workspace" in both comments

**Review by srinathnarayanan — DISMISSED** [review]

File: src/Explorer/Tree/ResourceTree.tsx (hunk 1 above)
Comment (swviswan) [inline src/Explorer/Tree/ResourceTree.tsx:400]: >context [](http://example.com/codeflow?start=53&length=7)

why are we disabling rename for temp workspace?

Comment (kcheekuri) [reply src/Explorer/Tree/ResourceTree.tsx]: Updated description in the bug.

File: src/Explorer/Tree/ResourceTree.tsx [hunk 2]
```diff
@@ -397,6 +397,11 @@ export const ResourceTree: React.FC<ResourceTreeProps> = ({ container }: Resourc
       },
     ];
 
+    //disallow renaming of temporary notebook workspace" in both comments
```
Comment (srinathnarayanan) [inline src/Explorer/Tree/ResourceTree.tsx:400]: this needs to be "disallow renaming of temporary notebook workspace"

File: src/Explorer/Tree/ResourceTreeAdapter.tsx [hunk 3]
```diff
@@ -808,6 +808,11 @@ export class ResourceTreeAdapter implements ReactAdapter {
       },
     ];
 
+    // For Temporary workspaces removing "Rename" in context menu.
```
Comment (srinathnarayanan) [inline src/Explorer/Tree/ResourceTreeAdapter.tsx:811]: this needs to be "disallow renaming of temporary notebook workspace"

**Review by srinathnarayanan — APPROVED** [review]

**Review by swviswan — APPROVED** [review]
:shipit:

**Pull request merged**